### ISSUE TITLE
Masking in 1.1.10

### ISSUE THREAD

**Issue by smajl** [platanus/angular-restmod#346]
@iobaixas Hi, so I have found out that my old code `someProperty: { mask: 'CU' }` does not work anymore and there were some changes in masking attributes in 1.1.10 (this works in 1.1.9). Are your masking tests passing?

If not, will there be a hotfix? If tests are passing and error has to be on my side, then how should I use the new "dynamic masks"? Maybe they would work for me at least... Thanks. :)

I am temporarily solving this by providing dummy `encode` function `encode: () => {}`

**Comment by iobaixas** [comment]
Hi @smajl,

Masking tests are passing, could you provide more detail on the model definition?

**Comment by smajl** [comment]
Well, here is the main model definition:
```javascript
export function Product(restmod) {
  'ngInject';
  return restmod.model('/products').mix('NotifyHooks', {
    categoryOfficialCId: { encode: parseInt },
    categorySaleCId:     { encode: parseInt },
    countryGId:          { encode: parseInt },
    destinationGId:      { encode: parseInt },
    markets: {
      decode: (val) => { return val ? _.compact(val.split(',')) : []; },
      encode: (val) => { return (val && val.length) ? val.join(',') : ''; }
    },
    productCapacities:   { mask: 'CU' },
    productDescriptions: { hasMany: 'Description', source: 'productDescriptions.data' },
    productImages:       { hasMany: 'Image', source: 'productImages.data' },
    productPeriods:      { mask: 'CU' },
    productPriceItems:   { mask: 'CU' },
    productSpecCId:      { encode: parseInt },
    productTags:         { hasMany: 'Tag', source: 'productTags.data' },
    vatPercentage:       { init: 100 },
    productUnits:        { hasMany: 'ProductUnit' }
  });
}
```
Properties `productCapacities`, `productPeriouds` and `productPriceItems` are properly masked in PUT and POST requests in 1.1.9, but not in 1.1.10.

**Comment by iobaixas** [comment]
@smajl, found the bug, I accidentally removed the mapping for the `mask` modifier and our specs only tested the underlying `attrMask` builder method.

A test has been added to prevent the keyword from being unmapped again and the fix has been published in 1.1.11.

Thanks for the heads up!

**Comment by smajl** [comment]
@iobaixas Thank you very much for the quick hotfix, much appreciated. :) I see that now both `mask` and `ignore` works in 1.1.11. ;)
